README.md

```diff
@@ -1,40 +1,119 @@
-# SPDX-Python-LicenseMatching
-Implement SPDX License Matching in Python. Project in CommunityBridge Linux Foundation 2020. 
-The Software Package Data Exchange (SPDX) specification is a standard format for communicating the components, licenses and copyrights associated with a software package. This Project is mainly based on implementing License Matching Techniques in Python. 
+# SPDX Python Library for LicenseMatching
+A Python Library to implement SPDX License Matching. This Project is a result of CommunityBridge Linux Foundation 2020 contribution by [@anshuldutt21](https://github.com/anshuldutt21/).
 
-## Check if 2 License Texts match
-License Texts are matched based on the [SPDX License Matching Guidelines](https://spdx.dev/license-list/matching-guidelines/). In order to implement License Matching between 2 License Texts, do the following steps.
+Home: https://github.com/anshuldutt21/spdx_python_licensematching/
+
+Issues: https://github.com/anshuldutt21/spdx_python_licensematching/issues
+
+# License
+[Apache-2.0](https://github.com/spdx/tools-python/blob/master/LICENSE)
+
+# Features
+
+*    An Interface which will check text against a license template using the license matching guidelines.
+*    An Interface which will check text and return all matching SPDX listed license ID's.
+*    An interface which takes 2 license texts as input and returns a boolean indicating if the 2 licenses match per the license matching guidelines.
+*    When there is not a match, a return value provided making it possible to describe where and why the license does not match.
```

Comment:
Change it to

*    An Interface which compares text against a license template using the license matching guidelines.
*    An Interface which returns all matching SPDX listed license ID's for any license text.
*    An interface which compares 2 license texts and returns a boolean indicating if the 2 licenses match per the license matching guidelines.
*  When there is no match, a return value is provided to describe where and why the license does not match.